doc/local-dev.md

```diff
@@ -0,0 +1,99 @@
+Your kubectl should be configured to use your minikube now and kubernetes should be available at `https://192.168.99.100:8443`.  
+Next install helm (see installation guides [here](https://github.com/kubernetes/helm/blob/master/docs/install.md)) and run `helm init` against your minikube. Check if tiller is successfully deployed with:  
+ `kubectl describe deployment tiller-deploy -n=kube-system`.
+
+A `secret.yaml` has to be prepared next, which contains all information for draughtsman to connect to your minikube and other entities:
+```
+service:
+  kubernetes:
+    incluster: false
+    address: https://192.168.99.100:8443
+    tls:
+      cafile: /minikube/apiserver.crt
+      crtfile: /minikube/apiserver.crt
+      keyfile: /minikube/apiserver.key
+  deployer:
+    environment: minikube
+    eventer:
+      github:
+        oauthtoken: your_token
+        organisation: your_org
+        projectlist: your_project, your_other_project
+    installer:
+      helm:
+        organisation: your_quay_org
+        password: your_quay_password
+        username: your_quay_username
+    notifier:
+      slack:
+        channel: your_channel
+  slack:
+    token: your_slack_token
+    ```
+Note that *all* of these values are mandatory! `Draughtsman` will not work if one or several of these values are missing!
+
+You now have to prepare your cluster itself now, a secret called `draughtsman-values-secret` and a configmap called `draughtsman-values-configmap` are necessary for draughtsman to run. Make sure a namespace called `draughtsman` exists.  
+ The configmap can look something like this:
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: draughtsman-values-configmap
+  namespace: draughtsman
+data:
+  values: |
+    Installation:
+      V1:
+        YOURORG:
+          YOURAPP:
+            Address:
+              Scheme: "something"
+              Host: "something"
+              ```
+Note that these are values match to the placeholders in your app ( see draughtsman-usage ).
+
+The secret should look like this:
+```
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: draughtsman-values-secret
+  namespace: draughtsman
+data:
+  values: SOMEBASE64
+```
+Where the base64 value should match this pattern (see draughtsman-usage):
+```
+Installation:
+  V1:
+    Secret:
+      APP:
+        Nginx:
+          Auth: SOMETHING
+      APP2:
+        Nginx:
+          Auth: SOMETHING
+```
+Now you can add both to your cluster with `kubectl apply -f filename`.
+
+#### Dev-cycle
+Once you've made a change in draughtsman first use `go build` to create the draughtsman binary.  
+Next up create the docker image: `docker build . --tag draughtsman-dev`  
+And then go ahead and start the container with some configs:  
+`docker run -v ~/.minikube:/minikube -v /your/folder:/var/run/draughtsmanconfig  draughtsman-dev daemon --config.dirs=/var/run/draughtsmanconfig --config.files=secret`  
```

Comment:
but no strong opinion, my thinking is it makes the guide easier, as you don't need to worry about explaining volume mounts